klayout/drc/rule_decks/lres.drc

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+  #================================================
+  #----------------N+ POLY RESISTOR----------------
+  #================================================
+
+  logger.info('Starting LRES derivations')
+  lres_poly = poly2.and(nplus).interacting(sab).interacting(res_mk)
+
+  # Rule LRES.1: Minimum width of Poly2 resistor. is 0.8µm
+  logger.info('Executing rule LRES.1')
+  lres1_l1 = lres_poly.width(0.8.um, euclidian)
+  lres1_l1.output('LRES.1', 'LRES.1 : Minimum width of Poly2 resistor. : 0.8µm')
+  lres1_l1.forget
+
+  # Rule LRES.2: Minimum space between Poly2 resistors. is 0.4µm
+  logger.info('Executing rule LRES.2')
+  lres2_l1 = lres_poly.isolated(0.4.um, euclidian)
+  lres2_l1.output('LRES.2', 'LRES.2 : Minimum space between Poly2 resistors. : 0.4µm')
+  lres2_l1.forget
+
+  # Rule LRES.3: Minimum space from Poly2 resistor to COMP.
+  logger.info('Executing rule LRES.3')
+  lres3_l1 = lres_poly.separation(comp, 0.6.um, euclidian).polygons(0.001.um).or(comp.not_outside(lres_poly))
+  lres3_l1.output('LRES.3', 'LRES.3 : Minimum space from Poly2 resistor to COMP.')
+  lres3_l1.forget
+
+  # Rule LRES.4: Minimum space from Poly2 resistor to unrelated Poly2. is 0.6µm
+  logger.info('Executing rule LRES.4')
+  lres4_l1 = lres_poly.separation(poly2.not_interacting(sab), 0.6.um, euclidian)
+  lres4_l1.output('LRES.4', 'LRES.4 : Minimum space from Poly2 resistor to unrelated Poly2. : 0.6µm')
+  lres4_l1.forget
+
+  # Rule LRES.5: Minimum Nplus implant overlap of Poly2 resistor. is 0.3µm
+  logger.info('Executing rule LRES.5')
+  lres5_l1 = lres_poly.enclosed(nplus, 0.3.um, euclidian).polygons(0.001.um)
+  lres5_l2 = lres_poly.not_outside(nplus).not(nplus)
+  lres5_l  = lres5_l1.or(lres5_l2)
+  lres5_l.output('LRES.5', 'LRES.5 : Minimum Nplus implant overlap of Poly2 resistor. : 0.3µm')
+  lres5_l1.forget
+  lres5_l2.forget
+  lres5_l.forget
+
+  # Rule LRES.6: Minimum salicide block overlap of Poly2 resistor in width direction. is 0.28µm
+  logger.info('Executing rule LRES.6')
+  lres6_l1 = lres_poly.enclosed(sab, 0.28.um)
+  lres6_l1.output('LRES.6', 'LRES.6 : Minimum salicide block overlap of Poly2 resistor in width direction. : 0.28µm')
+  lres6_l1.forget
+
+  # Rule LRES.7: Space from salicide block to contact on Poly2 resistor.
```

Comment:
should we add the spacing requirement to the comment (like we do in other rules?)

Reply:
@FaragElsayed2 Could you please update the comments for the rules that are missing values?

Reply:
@atorkmabrains Done